sdk/core/core-client-rest/test/getClient.spec.ts

```diff
@@ -25,15 +25,24 @@ describe("getClient", () => {
     });
 
     const apiVersion = "2021-11-18";
-    const client = getClient("https://example.org?api-version=1233321", { apiVersion });
-    const validationPolicy: PipelinePolicy = {
+    let client = getClient("https://example.org", { apiVersion });
+    let validationPolicy: PipelinePolicy = {
       name: "validationPolicy",
       sendRequest: (req, next) => {
         assert.include(req.url, `api-version=${apiVersion}`);
         return next(req);
       },
     };
 
+    client = getClient("https://example.org?api-version=1233321", { apiVersion });
```

Comment:
Can you create a separate test for what you have in lines 37 - 45? Line 37 and 38 will override 28 and 29 so the check in line 32 is never being executed

Reply:
sure will add tests